foundry.toml

```diff
@@ -12,6 +12,7 @@ fs_permissions = [{ access = "read-write", path = "./addresses/"}, { access = "r
 solc_version="0.8.21"
 ffi=true
 optimizer=false
+memory_limit = 100000000 # 33554432 is default limit
```

Comment:
Depending on how hard it is, adding manual memory management before merging could be nice. Otherwise this is a ticking time bomb, + it probably slows tests.

Reply:
I agree that this is not ideal. But as mentioned above, I think we should wrap up this test suite and park improvements for later, as we don't have much time and much to do.